Implement a method to check the existence of a workflow by name.

sadilchamishka opened the pull request:
### Proposed changes in this pull request
$subject

### Related Issues
- https://github.com/wso2/product-is/pull/24095

sadilchamishka commented on components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementService.java:
```diff
@@ -105,6 +105,19 @@ void addWorkflow(Workflow workflowDTO,
+        return false;
+    }
+
+
```
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/6781/commits/2b8281e16d2306f767f1962c913da585a491a5b7

Malith-19 commented on components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/util/SQLConstants.java:
```diff
@@ -187,6 +187,8 @@ public class SQLConstants {
     public static final String GET_WORKFLOW = "SELECT WF_WORKFLOW.WF_NAME, WF_WORKFLOW.DESCRIPTION, WF_WORKFLOW" +
```
can we rename this to `GET_WORKFLOW_BY_ID` since now you introduced a new query to get by name?

sadilchamishka replied:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/6781/commits/5f3fd473a78c9efd15df2cdb402c1b5649ffef10

sadilchamishka commented:
@Malith-19 I have updated the license headers of the classes we have moved to the framework by this commit https://github.com/wso2/carbon-identity-framework/pull/6781/commits/d1f74be67803fed21c8a55e4158d7f440ccda37d
